### DISCUSSION TITLE
IDEA sometimes does not display prompt code in code blocks

### DISCUSSION THREAD

**Discussion by mewhz** [community/community#15601] [Copilot Conversations] [answered]
At the beginning of use, it was displayed normally and after a while, there was no code reminder displayed in the code area, but the code can be displayed by opening the sidebar. After that, I restarted IDEA and he had no problem. There are more serious ones. I don't have a way to use Alt + [ to toggle the code hinting option in the code block, maybe it's my documentation shortcut understanding bug? thanks

**Comment by jansorg** [comment, marked as answer]
Thanks for reporting! 
Could you post the relevant snippets from your log file when this happens again,, please?

Here's how to enable debug logging for the Copilot plugin.
1. Open `Help > Diagnostic Tools > Debug Log Settings...`
2. Add a new line `#com.github.copilot` to enable debug logging for Copilot
3. Execute the action in your IDE which isn't working as expected
4. Open the log file with `Help > Show Log in ...`
5. Review the log file and post the relevant snippets here
